bin/feedbooks_monitor

```diff
@@ -1,18 +1,29 @@
 #!/usr/bin/env python
 """Update the content server with new open-access books from FeedBooks."""
+import logging
 import os
 import sys
 bin_dir = os.path.split(__file__)[0]
 package_dir = os.path.join(bin_dir, "..")
 sys.path.append(os.path.abspath(package_dir))
-from core.scripts import OPDSImportScript
-from feedbooks import FeedbooksOPDSImporter
+
 from core.model import DataSource
-for language in ['en', 'es', 'fr', 'it', 'de']:
-    url = "http://www.feedbooks.com/books/recent.atom?lang=%s" % language
-    OPDSImportScript(
-        url, 
-        None,
-        FeedbooksOPDSImporter,
-        immediately_presentation_ready=True,
-    ).run()
+
+from config import Configuration
+from scripts import OPDSImportScript
+from feedbooks import FeedbooksOPDSImporter
+
+import_script = OPDSImportScript(FeedbooksOPDSImporter, DataSource.FEEDBOOKS)
+base_opds_url = collection.external_account_id
+
+try:
+    for language in ['en', 'es', 'fr', 'it', 'de']:
+        # Update the OPDS url and run the script for each language.
+        import_script.collection.external_account_id = (
+            collection.external_account_id + language
+        )
+        print ('*' * 30) + collection.external_account_id + ('*' * 30)
+        import_script.run()
+        _db.commit()
```

Comment:
These should be five different collections since they're being imported from five different OPDS feeds.